project/Vagrantfile

```diff
@@ -65,13 +69,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         master.vm.box = "chef/centos-6.5"
         master.vm.provider "virtualbox" do |v|
             v.name = mastername
-            v.memory = 2048
-            v.cpus = 2
+            v.memory = @memory
+            v.cpus = @cpus
         end
         master.vm.hostname = mastername
         master.vm.network "public_network", bridge: 'en0: Wi-Fi (AirPort)'
+        #master.vm.network "private_network", type: "dhcp"
         master.vm.synced_folder "/opt/vagrant", "/opt/vagrant"
         master.vm.synced_folder "/space/software", "/space/software"
+        master.vm.synced_folder Dir.getwd, "/project/root"
```

Comment:
/vagrant mounts the folder where the VagrantFile is

> +        master.vm.synced_folder Dir.getwd, "/project/root"

This defaults to whenever the user is running the vagrant up command. Which
may not be the same.
Actually on my set up it's not:

I have my VagrantFile at ~/Workspace but I run the command under it's own
project folder:

~/Workspace/client1/

and that's where I have the different project.properties files

On Mon, Feb 23, 2015 at 12:04 PM, Geert <notifications@github.com> wrote:

> In project/Vagrantfile
> <https://github.com/grtjn/mlvagrant/pull/21#discussion_r25156109>:
>
> >          master.vm.synced_folder "/opt/vagrant", "/opt/vagrant"
> >          master.vm.synced_folder "/space/software", "/space/software"
> > +        master.vm.synced_folder Dir.getwd, "/project/root"
>
> This is not really necessary. The current dir is automatically mounted as
> /vagrant..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grtjn/mlvagrant/pull/21/files#r25156109>.
>